scripts/shell/change_app.toml.sh

```
@@ -0,0 +1,53 @@
+#
+# Targets: node0-node3 and seednode0
+# Changes:
+# - Enables telemetry collection
+# - Configures Prometheus retention
+# 
+#
+# Note: Run from a node with write access to all node directories
+##############################################################################
+
+# Navigate to primary node's config directory
+cd /data/node0/.treasurenetd/config/
```

Shellcheck (0.10.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)